common/src/device/provisioning/transport/ClientApiVersionHelper.cs

```diff
@@ -7,8 +7,5 @@ internal class ClientApiVersionHelper
     {
         public const string ApiVersionName = "api-version";
         public const string ApiVersion = "2017-08-31-preview";
```

Comment:
>2017-08-31-preview [](start = 42, length = 18)

Need to change to GA version.

Reply:
👍  dded TODO - I don't think another version is currently available on the service-side. /cc: @simonporter